src/itemCheckbox/index.tsx

```diff
@@ -11,6 +11,10 @@ const StyledItemCheckbox = styled(ItemCheckbox)`
     position: absolute;
     clip: rect(0, 0, 0, 0);
   }
+
+  &.kirk-item-checkbox--disabled {
+    cursor: default;
```

Comment:
Maybe remove the new class and use the prop instead (I received this same feedback when disabling icons):
```
& {
    cursor: ${props => (props.isDisabled ? 'default' : 'pointer')};
  }
```

Reply:
Thanks, it's way cleaner that way